docs/admin/config-guide/scim-config/user-config.md

```diff
@@ -403,3 +449,327 @@ Navigate to `Users` to open the users tab as shown in the image below:
 Janssen Server Configuration REST API exposes relevant endpoints for managing
 and configuring the OpenID Connect Client. Endpoint details are published in the [Swagger
 document](../../reference/openapi.md).
+
+### Get SCIM Client
```

Comment:
Since this is a separate tab in TUI, we should assign a separate document for it. Something like `docs/admin/config-guide/scim-config/scim-client-config.md` and that document should have content about how to configure SCIM using CLI, TUI and config API.

Reply:
> Since this is a separate tab in TUI, we should assign a separate document for it. Something like `docs/admin/config-guide/scim-config/scim-client-config.md` and that document should have content about how to configure SCIM using CLI, TUI and config API.

There are some info about this tab: https://docs.jans.io/v1.1.5/admin/scim/config/#configuration-management-using-tui